app/src/main/java/ru/dudar_ig/nasaimages/data/NasaApi.kt

```diff
@@ -0,0 +1,11 @@
+package ru.dudar_ig.nasaimages.data
+
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface NasaApi {
+    @GET("planetary/apod")
+    fun loadImage(@Query("api_key") apiKey: String,
```

Comment:
лучше всегда оставлять отступ между названием класса и его телом

```suggestion
interface NasaApi {

    @GET("planetary/apod")
```